cobra/test/test_io/test_notes.py

```diff
@@ -0,0 +1,33 @@
+from os.path import join
+
+import pytest
+
+import cobra
+from cobra.io import read_sbml_model, write_sbml_model
+from cobra.util.solver import set_objective
+
+
+def test_notes(tmp_path):
+    """Testing if model notes are written in SBML"""
+    path_to_file = join(str(tmp_path), "model_notes.xml")
+
+    # making a minimal cobra model to test notes
+    model = cobra.Model("e_coli_core")
+    model.notes["Remark"] = "...Model Notes..."
+    met = cobra.Metabolite("pyr_c", compartment="c")
+    model.add_metabolites([met])
+    met.notes["Remark"] = "I appear."
+    rxn = cobra.Reaction("R_ATPM")
+    model.add_reactions([rxn])
+    rxn.notes["Remark"] = "What about me?"
+    model.objective_direction = "max"
+    coefficients = {}
+    coefficients[rxn] = 1
+    set_objective(model, coefficients)
```

Comment:
I think the more straight forward way to set an objective is:

```suggestion
    model.objective = rxn
```